### PR TITLE
Enable more lint errors

### PR DESCRIPTION
**Pull request by simeg**

### CODE REVIEW

**Comment by simeg** [comment]
@hugovk Yeah, I looked at it and decided to not fix the problems but maybe you're right, we should just fix them. Will add to this PR.

File: SublimeText/Packages.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ The simplest method of installation is through the Sublime Text console. The con
 
 This snippet changes each version and older versions will return an error.
 
-To get the current versions code, go to https://packagecontrol.io/installation
+To get the current versions code, go to [their website](https://packagecontrol.io/installation)
```
Comment (kamleshkc2002) [inline SublimeText/Packages.md:9]: NIT: `their website` -> `packagecontrol.io`

**Comment by kamleshkc2002** [comment]
This is great. one small step for standardization, one great step for sanity! :D

**Comment by simeg** [comment]
Fixed your comments so I'm merging this

File: Heroku/README.md [hunk 2]
```diff
@@ -49,9 +49,17 @@ A cheat sheet for deployment:
 
 ```console
```
Comment (hugovk) [inline Heroku/README.md:50]: This still doesn't show output, so `sh` and no dollars would be better. The comments can stay.

But this can wait!

**Pull request merged**